internal/dataplane/parser/golden_test.go

```diff
@@ -280,8 +280,7 @@ func runParserGoldenTest(t *testing.T, tc parserGoldenTestCase) {
 
 		require.Equalf(t, string(goldenB), string(resultB),
 			"Golden file %s does not match the result. \n"+
-				"If you are sure the result is correct, run the test "+
-				"with the -update-golden flag to update the golden file: \n"+
+				"If you are sure the result is correct, update the golden file: \n"+
```

Comment:
I'd leave this as is.

> usage: go test [build/test flags] [packages] [build/test flags & test binary flags]

This works as intended when run as (mark the flag is provided after the package to be tested)

```
go test -v -run TestParser_GoldenTests ./internal/dataplane/parser -update
```

https://github.com/Kong/kubernetes-ingress-controller/blob/8a311ce62596819cfd2e4024a65d0b11a596f330/Makefile#L341-L343

Reply:
Ok, it seem that we forgot to adjust the comment. It's `update` not `-update-golden`.